ui/app/src/App.tsx

```diff
@@ -22,16 +21,14 @@ import Header from './components/Header';
 import Footer from './components/Footer';
 import { pluginRuntime } from './model/plugin-runtime';
 
-const queryClient = new QueryClient();
-
 function App() {
   const dashboard = useSampleData<DashboardResource>('dashboard');
   if (dashboard === undefined) {
     return null;
   }
 
   return (
-    <QueryClientProvider client={queryClient}>
```

Comment:
I moved this up to `index.tsx` since that's typically where those sorts of providers live (probably should have put it there to begin with).